docker-run.sh

```diff
@@ -100,30 +100,31 @@ fi
 docker start -a -i "$container"
 
 # Copy the output data back out of the container
-#echo "Copying traced data"
-#mkdir -p "$(dirname "$OUTPUT_JSON")"
-#docker cp "$container:/data/output.json" "$OUTPUT_JSON"
+echo "Copying out traced data..."
```

Comment:
Might be even better to say "Copying traced data from '$container:/data/output.json' to '$OUTPUT_JSON'"

Reply:
Update thanks!